index.html

```diff
@@ -830,9 +830,9 @@ <h3>JsonWebKey</h3>
 for an instance of a public key with a compound key identifier.
                 </p>
                 <p>
-The OPTIONAL `alg` property identifies the algorithm intended
-for use with the public key. Although optional, it is RECOMMENDED that `alg`
-be included, to avoid security issues that arise when using the same key with multiple
+The REQUIRED `alg` property identifies the algorithm intended
+for use with the public key, which is
```

Comment:
Sorry, wrong conjunction. `which` would apply `included` to `public key`, but it should be applied to ``` `alg` ```.
```suggestion
for use with the public key, and is
```